#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 792877a08ca7bc9d1377161dc9af374602a394cc Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Fri, 13 Apr 2018 15:45:54 +0900
Subject: [PATCH Libgpg-error] doc: Fix yat2m build for cross compilation.

* doc/Makefile.am: Fix target of yat2m-for-build.

--

GnuPG-bug-id: 3901
Fixes-commit: efc4769339d42a4a399c040c146cf4a29c02ea4f
Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 7439a49..76af5be 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -46,7 +46,7 @@ YAT2M_CMD = ./yat2m-for-build
 YAT2M_DEP = yat2m-for-build
 CLEANFILES += yat2m-for-build

-yat2m-for-build$(EXEEXT): yat2m.c
+yat2m-for-build: yat2m.c
	$(CC_FOR_BUILD) -o $@ $(srcdir)/yat2m.c
 else
 YAT2M_CMD = ./yat2m
--
2.11.0
